<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>regis</class>
 <widget class="QWidget" name="regis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>365</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>580</width>
    <height>365</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>580</width>
    <height>365</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>371</width>
     <height>71</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>方正粗黑宋简体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt; font-weight:600; color:#466731;&quot;&gt;注册一个账户&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>241</width>
     <height>241</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/prefer.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>121</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/Apple.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-10</y>
     <width>611</width>
     <height>381</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::MarkdownText</enum>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/yellow.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>50</y>
     <width>611</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/2222.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/xiaohui.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>320</y>
     <width>611</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/2222.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_account">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>157</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入您的姓名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_passwd">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>200</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>设置密码最少六位</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_passwd2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>243</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>再次输入密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_signup_user">
   <property name="geometry">
    <rect>
     <x>298</x>
     <y>290</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>注册用户</string>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/zheceaaa.png</normaloff>:/zheceaaa.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="lb_passwd2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>243</y>
     <width>64</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正姚体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确认密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_passwd">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>200</y>
     <width>56</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正姚体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string> 密  码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_signup_superviser">
   <property name="geometry">
    <rect>
     <x>384</x>
     <y>290</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>注册管理员</string>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/zheceaaa.png</normaloff>:/zheceaaa.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_cancel">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>290</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/canso.png</normaloff>:/canso.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="lb_account">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>157</y>
     <width>56</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正姚体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string> 姓  名</string>
   </property>
  </widget>
  <zorder>label_3</zorder>
  <zorder>label_5</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>label</zorder>
  <zorder>le_account</zorder>
  <zorder>le_passwd</zorder>
  <zorder>le_passwd2</zorder>
  <zorder>bt_signup_user</zorder>
  <zorder>lb_passwd2</zorder>
  <zorder>lb_passwd</zorder>
  <zorder>label_2</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>label_4</zorder>
  <zorder>bt_signup_superviser</zorder>
  <zorder>bt_cancel</zorder>
  <zorder>lb_account</zorder>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
